<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>287</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Block Adder M7</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="idLbl">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="connectionLbl">
     <property name="text">
      <string>[C-&gt;S]</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Signature</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="sigEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Packet ( To filter use (*), which will be ignored)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="packetEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="addBtn">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
